### ISSUE TITLE
Missing 508 mapping on some rules

### ISSUE THREAD

**Issue by skurapati** [dequelabs/axe-core#156]
Rules like data-table, frame-title etc. are missing section 508 mapping in tags. data-table can be mapped to Para-G of Section 508, frame-title can be mapped to Para-I. There might be other rules as well that are missing section 508 mapping. These two are just examples.

**Comment by WilcoFiers** [comment]
We've created a new mapping for sec508. This will be available in the next major release of axe-core. Thanks for contributing!
